website/content/en/preview/concepts/_index.md

```diff
@@ -6,169 +6,105 @@ description: >
+While using Kubernetes well-known labels, the NodePool can set some values that are specific to the cloud provider. So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
 
-{{% alert title="Note" color="warning" %}}
-Karpenter publishes Kubernetes events to the node for all events listed above in addition to __Spot Rebalance Recommendations__. Karpenter does not currently support cordon, drain, and terminate logic for Spot Rebalance Recommendations.
-{{% /alert %}}
+### Kubernetes Cluster Autoscaler
 
-### Kubernetes cluster autoscaler
-Like Karpenter, [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is
-designed to add nodes when requests come in to run pods that cannot be met by current capacity.
-Cluster autoscaler is part of the Kubernetes project, with implementations by most major Kubernetes cloud providers.
-By taking a fresh look at provisioning, Karpenter offers the following improvements:
+Like Karpenter, [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is designed to add nodes when requests come in to run pods that cannot be met by current capacity. Cluster autoscaler is part of the Kubernetes project, with implementations by most major Kubernetes cloud providers. By taking a fresh look at provisioning, Karpenter offers the following improvements:
 
-* **Designed to handle the full flexibility of the cloud**:
-Karpenter has the ability to efficiently address the full range of instance types available through AWS.
-Cluster autoscaler was not originally built with the flexibility to handle hundreds of instance types, zones, and purchase options.
+* **Designed to handle the full flexibility of the cloud**: Karpenter has the ability to efficiently address the full range of instance types available through AWS. Cluster autoscaler was not originally built with the flexibility to handle hundreds of instance types, zones, and purchase options.
 
-* **Group-less node provisioning**: Karpenter manages each instance directly, without use of additional orchestration mechanisms like node groups.
-This enables it to retry in milliseconds instead of minutes when capacity is unavailable.
-It also allows Karpenter to leverage diverse instance types, availability zones, and purchase options without the creation of hundreds of node groups.
+* **Group-less node provisioning**: Karpenter manages each instance directly, without use of additional orchestration mechanisms like node groups. This enables it to retry in milliseconds instead of minutes when capacity is unavailable. It also allows Karpenter to leverage diverse instance types, availability zones, and purchase options without the creation of hundreds of node groups.
```

Comment:
Thoughts on removing this? With Drift fully implemented, this is less true, and I think we've been moving away from this for a while.

Reply:
How about just renaming the section to something like "Quick node provisioning"